src/share/classes/com/sun/javatest/regtest/tool/i18n.properties

```diff
@@ -171,6 +171,10 @@ help.main.ignore.error.desc=(Default.) Execute the actions up to the @ignore tag
     then give an "Error" result.
 help.main.ignore.run.desc=Run the test, as though the @ignore tag were not present.
 help.main.l.desc=List the tests that would be executed instead of executing them.
+help.main.mw.arg=Man wrapper argument
+help.main.mw.desc=Man wrapper desc
+help.main.mwp.arg=Man wrapper argument
+help.main.mwp.desc=Man wrapper desc
```

Comment:
possible spelling errors: should "Man wrapper" be "Main" -- if not, what is a "Man Wrapper" ?

Reply:
fixed